pandas/tests/extension/test_integer.py

```diff
@@ -142,6 +142,11 @@ def _check_op(self, s, op, other, op_name, exc=NotImplementedError):
                 # combine method result in 'biggest' (int64) dtype
                 expected = expected.astype(s.dtype)
                 pass
+            if op_name == "__rpow__":
+                # TODO: https://github.com/pandas-dev/pandas/issues/29997
```

Comment:
Should `1 ** pd.NA` be 1 or `NA`? See https://github.com/pandas-dev/pandas/issues/29997.

R returns 1. Julia returns missing.